grabbit/src/main/groovy/com/twcable/grabbit/client/batch/ClientBatchJob.groovy

```diff
@@ -179,6 +180,7 @@ class ClientBatchJob {
                     "${PORT}"             : serverBuilder.port,
                     "${USERNAME}"         : credentialsBuilder.username,
                     "${PASSWORD}"         : credentialsBuilder.password,
+                    "${EXCLUDE_PATHS}"    : pathConfiguration.excludePaths.join("<sp>"),
```

Comment:
@masroormohammed I'm not familiar with this notation "<sp>".  Does this represent a space?